demos/using-dev-build/README.md

```diff
@@ -1,16 +1,68 @@
 # Integrating with Internet Identity
 
-This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors).
+This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library. This is a standalone project that you can copy to your own project.
```

Comment:
I would put the sentence "This is a standalone project that you can copy to your own project." in a new line. But this is a personal preference.